src/mcc-group.ts

```diff
@@ -60,4 +60,31 @@ export class MccGroupApi {
     }
     return { success: !resp?.payload?.errorCode, mccGroup: { ...resp.payload } }
   }
+
+  /*
+   * Function to take a MCC Group token Id, send that to Marqeta, and return
+   * the MCC Group information.
+   */
+  async get(token: string): Promise<{
+    success: boolean,
+    mccGroup?: MccGroup,
+    error?: MarqetaError,
+  }> {
+    const resp = await this.client.fire('GET',
+      `mccgroups/${token}`,
+      undefined,
```

Comment:
If you don't need query params or a body, then you can drop the `undefined` as well, and just use:

```typescript
     const resp = await this.client.fire('GET',
       `mccgroups/${token}`,
     }
```